ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -235,3 +244,14 @@ Set Expected Items Based On RHODS Type    # robocop: disable
     Set Suite Variable    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}    ${EXPECTED_ITEMS_FOR_PROVIDER_TYPE}
     Set Suite Variable    ${EXPECTED_ITEMS_FOR_COMBINATIONS}    ${EXPECTED_ITEMS_FOR_COMBINATIONS}
 
+Validate App Name Is Present On Each Tile
+    [Documentation]    Check that each Resource tile contains
+    ...    the name of the application
+    ${elements} =    Get WebElements    //article[contains(@class, 'pf-c-card')]//div[@class="pf-c-content"]//small
+    ${len}=    Get Length    ${elements}
+    FOR    ${counter}    IN RANGE    ${len}
+        ${name}=    Get Text    ${elements}[${counter}]
+        should contain    ${name}    by
+        ${length} =	Get Length    ${name}
+        Should Be True	${length} > 5
```

Comment:
You can also use `Should Start With` to verify that the app name  starts with _by_ and also use [Remove String](https://robotframework.org/robotframework/latest/libraries/String.html#Remove%20String) to get it removed from the  string and obtain the app name